index.html

```diff
@@ -0,0 +1,557 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New York Times</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="https://kit.fontawesome.com/7bc3c221d9.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <!-- NAVBAR -->
+    <nav class="navbar">
+        <div class="left">
+            <button class="icon-btn"><i class="icon fas fa-bars"></i></button>
+            <button class="icon-btn"><i class="icon fas fa-search"></i></button>
+            <a href="https://www.nytimes.com/section/science/space">SPACE & COSMOS</a>
```

Comment:
- [ ] The link should not be blue

- [ ] There is a white margin overflowing

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91743532-57d9db00-eb8e-11ea-83c0-905105e8a9cb.png) | ![image](https://user-images.githubusercontent.com/61479648/91743613-76d86d00-eb8e-11ea-9168-f0473acd01b8.png) |